ibtk/src/solvers/impls/fortran/patchsmoothers2d.f.m4

```diff
@@ -878,6 +879,7 @@ c
      &     use_harmonic_interp,
      &     red_or_black)
 c
+      use average
       implicit none
 c
 c     Functions.
```

Comment:
in particular, on new line 887 we are still declaring h_avg and a_avg in the old way.